packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -17,12 +17,14 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @custom:value GAS_CONFIG           Represents an update to txn fee config on L2.
     /// @custom:value GAS_LIMIT            Represents an update to gas limit on L2.
     /// @custom:value UNSAFE_BLOCK_SIGNER  Represents an update to the signer key for unsafe
+    /// @custom:value FEE_SCALARS_ECOTONE  Represents an update to fee scalars on L2 after ecotone.
     ///                                    block distrubution.
     enum UpdateType {
         BATCHER,
         GAS_CONFIG,
         GAS_LIMIT,
-        UNSAFE_BLOCK_SIGNER
+        UNSAFE_BLOCK_SIGNER,
+        FEE_SCALARS_ECOTONE
```

Comment:
went back and forth around keeping the old `GAS_CONFIG` naming vs this `FEE_SCALARS` one - if using the Fee Scalar naming instead, we could techincally omit the "ecotone" suffix throughout - open to thoughts